core/conversation_tree.py

```
@@ -17,15 +17,31 @@ def __init__(self, provider: str = "anthropic"):
+        """
+        branches = []
         queue = [self.root]
+        visited = {self.root.id}
+        
         while queue:
             node = queue.pop(0)
             if node.branch_name:
-                names.add(node.branch_name)
-            queue.extend(node.children)
-        return names
+                 is_current_branch = False
```

Ruff (0.14.10)

Local variable `is_current_branch` is assigned to but never used

Remove assignment to unused variable `is_current_branch`

(F841)